Lab-meetings.md

```diff
@@ -0,0 +1,347 @@
+# Lab Meetings
+
+As the lab grows, the importance of sharing technical skills, domain expertise and tips and tricks for working effectively grows with it.
+Lab meetings are a space for all members of the group to come together and learn from each other.
+
+The meetings are held at the Turing Institute on Thursdays at 3pm.
+Remote participation will always be available and is strongly encouraged for lab members who are not physically located at the Institute that day.
+The [Whitaker Lab calendar](https://calendar.google.com/calendar/embed?src=7nar31c6ni4esif8fn1881kgds%40group.calendar.google.com) shows the times of the meetings, and when meetings are cancelled because they conflict with a relevant event that multiple lab members may wish to attend.
+
+**Every lab member is expected to participate in the lab meetings.**
+This doesn't mean you have to attend every one, but it does mean that you are expected to complete the HackMD with your attendance or apologies, and to add something to share and a question to ask.
+
+This document will talk you through the expectations for participation [before](#preparing-for-lab-meeting), [during](#participation-during-lab-meeting) and [after](#publishing-a-summary-blog-post-after-lab-meeting) lab meeting.
+There is also a long [motivation](#motivation) section about why we've designed the meetings this way.
+
+Lab meetings are _very_ useful for me (Kirstie), but I hope they are valuable for you too.
+I am always interested to hear of any changes that you'd like to make.
+
+## Participation in lab meeting
+
+### Preparing for lab meeting
+
+In advance of each week's lab meeting, the chair will send an email with a link to a HackMD note.
+
+HackMD is a realtime, multi-platform collaborative markdown note editor.
+Everyone is able to edit the text in [markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) when you click the the `edit` page, and HackMD renders the raw input as a nicely formatted website in the `view` rendering.
+You can see the raw markdown and the rendered page side by side by clicking the `both` button.
+You can read more about HackMD at [https://hackmd.io/s/features](https://hackmd.io/s/features).
+
+The template for each lab meeting is below, and you can read more about why we have this format in the [motivation](#motivation) section at the end of this page.
+
+When you receive the HackMD link, please add your name under **Attendees** or **Apologies**.
+This lets the chair know that you've received the email and acknowledges that week's meeting.
+It also means the chair knows who to wait for at the start of the meeting.
+
+In advance of the meeting, please also complete the sections on **What do you want the lab members to know about?** and **What would you like lab members' opinion(s) on?**
```

Comment:
So a lab meeting is about sharing what we are _individually_ working on, as opposed to setting a group project or sharing progress on something we are _collectively_ working on?

Reply:
Its a space to build connections between members of the group.

From my point of view it's really beneficial for folks to share skills and knowledge to enhance each other's separate projects. As I'm involved in almost all of them I can see clear overlaps and where skills and outputs can be re-used across different projects.....lab meeting is a space to try to make those clearer too.

I think starting a new project is brittle - most members of the group are flat out on their current projects so adding a shared one is likely to fail when everyone gets busy, and is going to be hard to find overlap for everyone's interests. Individual lab meetings could be used to work through specific skill and knowledge shares though!

Does that make sense? Is there a good place to add this clarification to the guidance?